package.json

```diff
@@ -70,7 +70,9 @@
   },
   "peerDependencies": {
     "plotly.js": ">1.34.0",
-    "react": ">0.13.0"
+    "react": ">0.13.0",
+    "flyd": ">=0.2.8",
+    "ramda": ">=0.28.0"
```

Comment:
`flyd` and `ramda` seem like they'd be better as `dependencies` rather than `peerDependencies` - and then let's change from `>=` to `^`

Reply:
Are 2 news deps really required?
`ramda` seems to be use just to write this code in the functional style.
The author himself said in the related issue that it could be replaced with 
```
function getSizeForLayout(entries) {
    const { width, height } = entries[0].contentRect;
    return { layout: { width, height } };
}
```

Reply:
Good question @floriancargoet - `ramda` has a couple of other uses in this PR (and we do use it a good deal elsewhere at Plotly), but to my mind your `getSizeForLayout` rewrite is more readable than the full functional version.

Reply:
While yes `getSizeForLayout` can be written easily without `ramda` (happy to do this), the main reason it was included was for `mergeDeepRight` which is a relatively complex function to implement. 
We could go for an independent implementation but my argument would be that `ramda` is relatively small and compatible with tree shaking